Client/SafeBrowsingPages/Phishing.html

```diff
@@ -0,0 +1,216 @@
+                left: 50%;
+                max-width: 650px;
+                -ms-transform: translate(-50%, -50%);
+                transform: translate(-50%, -50%);
+            }
+        }
+    </style>
+    <script>
+        function toggleDetails() {
+            var element = document.getElementById("details");
+            var arrow = document.getElementById("arrow");
+            if (element.classList.contains("hidden")) {
+                element.classList.remove("hidden");
+                arrow.classList.replace("down", "up");
+            } else {
+                element.classList.add("hidden");
+                arrow.classList.replace("up", "down");
+            }
+        }
+
+        function backToSafety() {
+            webkit.messageHandlers.InterstitialPageHandler.postMessage("go_back");
+        }
+
+        function proceedUnsafe() {
+            webkit.messageHandlers.InterstitialPageHandler.postMessage("proceed_unsafe");
+        }
+    </script>
+</head>
+
+<body class="background content">
+    <div class="container post">
+        <svg class="icon" xmlns="http://www.w3.org/2000/svg">
+            <g fill="rgb(250, 251, 251)">
+                <path
+                    d="M58.584 59.167H5.414a2.423 2.423 0 01-2.116-3.589L29.86 7.158c.848-1.544 3.385-1.544 4.232 0l26.355 48.045a2.42 2.42 0 01-1.862 3.964zM31.975 13.35L9.498 54.325h44.954L31.975 13.35zm0 11.922a2.418 2.418 0 012.416 2.421V39.8a2.42 2.42 0 01-2.416 2.421 2.417 2.417 0 01-2.413-2.421V27.694a2.416 2.416 0 012.413-2.42zm1.721 20.069c.434.46.7 1.09.7 1.72 0 .63-.266 1.258-.724 1.718a2.437 2.437 0 01-1.69.703 2.486 2.486 0 01-1.714-.703 2.433 2.433 0 01-.702-1.718c0-.63.242-1.26.702-1.72.891-.896 2.535-.896 3.428 0z" />
+            </g>
+        </svg>
+
+        <p class="title">
+            Deceptive site ahead
+        </p>
+
+        <p class="description">
+            Attackers on testsafebrowsing.appspot.com may trick you into doing something dangerous like installing
+            software or revealing your personal information (for example, passwords, phone numbers, or credit cards).
+            <a class="description" href="https://support.google.com/chrome/answer/99020">Learn more</a>
+            <br /><br />
+        </p>
+
+        <div class="navigationButtons">
+            <button class="leftNavButton leftNavButtonTitle" onclick="toggleDetails();">More details <i id="arrow"
+                    class="arrow down"></i></button>
+            <button class="rightNavButton rightNavButtonTitle" onclick="backToSafety();">Back to safety</button>
+        </div>
+
+        <p id="details" class="moreDetails hidden">
+            Google Safe Browsing recently detected phishing on testsafebrowsing.appspot.com. Phishing sites pretend to
+            be
+            other websites to trick you.
+            <br /><br />
+            You can report a detection problem or, if you understand the risks to your security, <a class="moreDetails"
```

Comment:
"report a detection problem" should be a link to https://safebrowsing.google.com/safebrowsing/report_error/?url=https://example.com (again, with `https://example.com` replaced with the full URL, url-escaped).